provider/ethereum/token/client.go

```diff
@@ -27,17 +28,18 @@ import (
 	"github.com/tdewolff/minify/v2/minify"
 	"github.com/tidwall/gjson"
 	"github.com/vincent-petithory/dataurl"
-	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	defaultCacheDuration = 24 * time.Hour
```

Comment:
I am okay with this not being loaded from env, but is 24H too short? I don't think the majority ever updates metadata.

Reply:
Changed to 7 days.